probe/endpoint/nat.go

```diff
@@ -50,7 +50,10 @@ func toMapping(f Flow) *endpointMapping {
 // ApplyNAT duplicates Nodes in the endpoint topology of a
 // report, based on the NAT table as returns by natTable.
 func (n NATMapper) ApplyNAT(rpt report.Report, scope string) {
-	n.WalkFlows(func(f Flow) {
+	if n.Conntracker == nil {
+		return
+	}
```

Comment:
NATMapper cannot be nil anymore as it's a value-type. Further, I'd rather not use nil as an ersatz Option type. It feels like the right way to do this isn't to make nil checks deep in component logic, but rather to assume all dependencies are valid, and provide no-op implementations when they may be disabled, or fail to be constructed.